### ISSUE TITLE
Nushell compatibility

### ISSUE THREAD

**Issue by schlich** [olzhasar/pytest-watcher#37]
Hello, thank you for the wonderful package.  It has really been helping me w/my TDD workflow.

I use [nushell](https://www.nushell.sh/) as my default shell environment whenever possible, however pytest-watcher shows some quirky behavior when run via nushell (note that nushell is not POSIX compliant so its interop compatibility is a common issue).

Things mostly run perfectly (nice!) except i get an error message display after the results for each test run.
 (filepaths slightly adjusted to redact personal info).  note this error doesnt seem to affect the behavior of the test run.

```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "~/.local/share/uv/python/cpython-3.11.9-macos-aarch64-none/lib/python3.11/threading.py", line 1045, in _bootstrap_inner
    self.run()
  File "~/data-integration/.venv/lib/python3.11/site-packages/watchdog/observers/api.py", line 213, in run
    self.dispatch_events(self.event_queue)
  File "~/data-integration/.venv/lib/python3.11/site-packages/watchdog/observers/api.py", line 391, in dispatch_events
    handler.dispatch(event)
  File "~/data-integration/.venv/lib/python3.11/site-packages/pytest_watcher/event_handler.py", line 52, in dispatch
    if self._is_event_watched(event):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/data-integration/.venv/lib/python3.11/site-packages/pytest_watcher/event_handler.py", line 49, in _is_event_watched
    return match_any_paths(paths, self.patterns, self.ignore_patterns)
TypeError: match_any_paths() takes 1 positional argument but 3 were given
```

If addressing this would be difficult i would understand closing this as a "wont fix" but if the fix is not too bad I'd be happy to help contribute.

Thanks!

**Comment by schlich** [comment]
Hm, these issues are no longer happening on my machine after running some updates.  closing unless it becomes an issue again.  thanks anyways!

**Comment by olzhasar** [comment]
Hey,
The error you mentioned was caused by a compatibility issue with the latest `watchdog` release, which introduced some breaking changes. This was fixed in the v0.4.3 of `pytest-watcher`.
I am not sure how nushell works, but generally, aliases should be picked once you activate the virtual environment where `pytest-watcher` is installed.
